src/index.js

```diff
@@ -1 +1,21 @@
 'use strict';
+
+import express from 'express';
+import cors from 'cors';
+
+import 'dotenv/config';
+import { roomRouter } from './routes/room.route';
+
+const PORT = process.env.PORT || 3005;
+const app = express();
+
+app.use(express.json());
+app.use(cors());
+
+app.use('/message', messageRouter);
```

Comment:
The `messageRouter` is used here but not imported. Ensure you import `messageRouter` from the correct path, similar to how `roomRouter` is imported.